src/GuppyDataExplorer/ExplorerFilter/index.jsx

```diff
@@ -38,6 +38,10 @@ class ExplorerFilter extends React.Component {
         }
       }
     }
+    return null;
```

Comment:
to address a warning found in console while testing using dev.html.
`Warning: ExplorerFilter.getSnapshotBeforeUpdate(): A snapshot value (or null) must be returned. You have returned undefined.`